docs/guides/extensions/panels-display-data-vonage.md

```diff
@@ -0,0 +1,769 @@
+
+```js
+{
+	field: 'include_message',
+	name: 'Include Message',
+	type: 'boolean',
+	meta: {
+		interface: 'boolean',
+		width: 'half',
+	},
+	schema: {
+		default_value: false,
+	}
+},
+```
+
+Lastly, add the option to limit the messages to a specific state such as delivered or failed. The default option is
+`Any`:
+
+```js
+{
+	field: 'status',
+	name: 'Status',
+	type: 'string',
+	schema: {
+		default_value: 'any',
+	},
+	meta: {
+		width: 'half',
+		interface: 'select-dropdown',
+		options: {
+			choices: [
+				{ text: 'Any', value: 'any' },
+				{ text: 'Delivered', value: 'delivered' },
+				{ text: 'Expired', value: 'expired' },
+				{ text: 'Failed', value: 'failed' },
+				{ text: 'Rejected', value: 'rejected' },
+				{ text: 'Accepted', value: 'accepted' },
+				{ text: 'buffered', value: 'buffered' },
+				{ text: 'Unknown', value: 'unknown' },
+				{ text: 'Deleted', value: 'deleted' }
+			]
+		}
+	}
+},
+```
+
+After the `options` section, there is the ability to limit the width and height of the panel. Since this panel will hold
+a lot of data, set these to `24` for the width and `18` for the height:
+
+```js
+minWidth: 24,
+minHeight: 18,
+```
+
+The output of these options will look like this:
+
+<img src="https://marketing.directus.app/assets/053298df-a5ff-4fd5-85ec-ff0f78bb64ca.png" alt="Form shows product type dropdown, direction dropdown, date range dropdown, included message checkbox, and status dropdown." style="padding: 2px 12px;">
+
+## Prepare the View
+
+Open the `panel.vue` file and you will see the starter template and script. Skip to the script section and import the
+following packages:
+
+```js
+import { useApi } from '@directus/extensions-sdk';
+import { adjustDate } from '@directus/shared/utils';
+import { formatISO, formatDistanceToNow, parseISO } from 'date-fns';
+import { ref, watch } from 'vue';
+```
+
+In the `props`, `showHeader` is one of the built-in properties which you can use to alter your panel if a header is
+showing. Remove the text property and add all the options that were created in the previous file:
+
+```js
+props: {
+	showHeader: {
+		type: Boolean,
+		default: false,
+	},
+	type: {
+		type: String,
+		default: '',
+	},
+	direction: {
+		type: String,
+		default: '',
+	},
+	range: {
+		type: String,
+		default: '',
+	},
+	include_message: {
+		type: Boolean,
+		default: false,
+	},
+	status: {
+		type: String,
+		default: '',
+	},
+},
+```
+
+After the `props`, create a `setup(props)` section and create the variables needed:
+
+```js
+
+setup(props){
+	const api = useApi();
+	const activity_data = ref([]);
+	const now = ref(new Date());
+	const is_loading = ref(true);
+	const has_error = ref(false);
+},
+
+```
+
+Create a `fetchData` function that will use the information provided to construct the query parameters and perform the
+API query. The response is written to the `activity_data` constant.
```

Comment:
~~constant~~ ref. You can't write to a constant as it's constant 🙂

Reply:
Ah yes I thought I caught these - Tim wrote constants and I changed to variable. Missed this one.